site/app/templates/forum/ThreadPostForm.twig

```diff
@@ -150,10 +150,10 @@
     {% endif %}
     <span class="inline-block right-zero position-absolute">
         {% if show_anon %}
-            <label for="Anon">Anonymous (to class)?</label> <input type="checkbox" class="thread-anon-checkbox" id="{{ anon_edit_post_id | default('thread_post_anon') }}" name="Anon" value="Anon" data-ays-ignore="true"/>
+            <label>Anonymous (to class)?</label> <input type="checkbox" class="thread-anon-checkbox" id="{{ anon_edit_post_id | default('thread_post_anon') }}" aria-label="Anonymous (to class)?" name="Anon" value="Anon" data-ays-ignore="true"/>
```

Comment:
When we want to display the label visually, for non screenreaders (like here), using the `<label>` tag with the `for=""` attribute is better. Similarly, you could also follow the pattern `<label><input /></label>` to achieve similar results. Using semantic HTML is always better than using ARIA attributes.

Reply:
Taking a look, thanks!